### DISCUSSION TITLE
Dominio accesso a google

### DISCUSSION THREAD

**Discussion by luigirotondi** [iisgiua/giuaschool#730] [Richieste di aiuto]
Salve, come si può modificare il dominio di accesso tramite google da (@giua.edu.it) al dominio del nostro istituto? 
![Screenshot_20240620_190756_Chrome](https://github.com/iisgiua/giuaschool/assets/159053557/dc296346-2767-43a8-b1e4-e2534beaac13)
@trinko

**Comment by trinko** [comment]
Assicurati di aver inserito il dominio del tuo istituto nel parametro **id_provider_dominio**.

Quindi esegui l'ultimo aggiornamento che contiene una correzione per la pagina di login.
